qa/rest-compat-tests/build.gradle

```diff
@@ -20,3 +20,5 @@ integTest.dependsOn(copyRestTestsResources)
 dependencies {
   compile project(':test:framework')
 }
+
+test.enabled = false
```

Comment:
note - this line will likely conflict with [this line #53228](https://github.com/elastic/elasticsearch/pull/53228/files#diff-02871ed50004171fe02cef11d5b87e96R25)

Reply:
I have only disabled it temporarily to see if this will try to continue the build